Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/Constants.java

```diff
@@ -0,0 +1,112 @@
+    public static final int US_DIGITAL_TICKS_PER_ROTATION = 1440;   // For odometry wheels
+    public static final double MM_PER_INCH = 25.4;
+    public static final double IN_FIELD_SIZE = 140.94;  // Not 144" due to interlocking pieces being cut off on field edge.
+    //---------------------------------------------------------------
+
+
+    // Tolerances----------------------------------------------------
+    public static final double ANGLE_TOLERANCE_DEG =  1.0;
+    public static final double POSITION_TOLERANCE_IN = 0.5;
+    public static final double OPENCV_TOLERANCE_PIX = 0.5;
+    public static final double LIFT_MOTOR_TOLERANCE_ENC_TICKS = 30; // Tolerance for automatic lift movements
+    //---------------------------------------------------------------
+
+
+    // Robot specs---------------------------------------------------
+    public static final double WHEEL_DIAMETER_IN = 4.0;         // 4 inch diameter wheel
+    public static final double SPROCKET_RATIO = 16.0 / 16.0;    // Driven to driving sprocket teeth
+    public static final double IN_PER_ANDYMARK_TICK = (Math.PI * WHEEL_DIAMETER_IN) / (0.5 * AM_40_TICKS_PER_ROTATION * SPROCKET_RATIO);    // Num inches per tick of drive motors
+    //---------------------------------------------------------------
+
+
+    // Drive mode constants------------------------------------------
+    public static final double SLOW_MODE_T_FACTOR = 0.3;        // Slow mode multipliers
+    public static final double SLOW_MODE_R_FACTOR = 0.3;
+    public static final double T_FACTOR = 1.0;                  // Normal multipliers
+    public static final double R_FACTOR = 1.0;
+    public static final double MAX_NAV_ROT_POWER = 0.3;         // Max rotation power when navigating
+    public static final double AUTO_SEARCH_TURN_POWER = 0.2;    // Power when looking for image targets
+    //---------------------------------------------------------------
+
+
+    // Movement control constants------------------------------------
+    public static final double MINIMUM_DRIVE_POWER = 0.08;
+    public static final double MAX_DRIVE_POWER = 1.0;
+    public static final double MINIMUM_TURNING_POWER = 0.02;    // todo Too small?
+    // Constants for adjusting powers that are proportional to angle and position differences
+    public static final double TURNING_POWER_FACTOR = 0.01;
+    public static final double DRIVE_POWER_FACTOR = 0.04;
+    //---------------------------------------------------------------
+
+
+    // todo Implement I and D terms; may need to adjust for new chassis
+    // PID loop constants--------------------------------------------
+    public static final double ROTATION_P = TURNING_POWER_FACTOR;
+    public static final double ROTATION_I = 0.0;
+    public static final double ROTATION_D = 0.0;
+    public static final double TRANSLATION_P = DRIVE_POWER_FACTOR;
+    public static final double TRANSLATION_I = 0/*0.0000005*/;
+    public static final double TRANSLATION_D = 0/*0.1*/;
```

Comment:
Why are these numbers commented out?

Reply:
It is because the I and D values need to be experimental determined

Reply:
Anything inside /* */ is a comment

Reply:
I can remove those